packages/echo-base/src/errors/randomHelper.ts

```diff
@@ -0,0 +1,3 @@
+export const randomId = function () {
```

Comment:
Would `generateRandomId()` from `echoUtils` work instead of this? Just to avoid duplicated features.

Reply:
echo-base shouldn't have dependencies to external libraries (if it can be avoided). Then it's better to duplicate smaller code bits than adding a dependency.

Reply:
Ah, true: we could end up in an update deadlock o___O